aedttest/simulation_data.py

```diff
@@ -226,14 +226,12 @@ def extract_data(desktop, project_dir, project_name, design_names):
 
         if not analyze_success:
             logger.error("design {} 'analyze_all' failed".format(design_name))
-
             error_messages = app.logger.get_messages(project_name, design_name, level=1, aedt_messages=True)
-            message = str(error_messages).replace("[error]", "")
-            message = "{}: {}".format(design_name, message)
-            logger.error(message)
-            PROJECT_DICT["error_exception"].append(message)
-
-            continue
+            messages = error_messages.design_level + error_messages.project_level + error_messages.global_level
```

Comment:
Need to test this first. For example, design1 and design2 both fail. What would be the project and global message?

Reply:
It depends on the type of error, typically in the global message, there is also interesting info about the error.